target/maven-archiver/pom.properties

```diff
@@ -1,5 +1,5 @@
 #Generated by Maven
```

Comment:
i think this file can be removed. not sure why it always adds itself on PRs (i have the same problem)

Reply:
Thanks